src/algorithms/tracking/ActsTracksToTrajectoriesHelper.h

```diff
@@ -0,0 +1,59 @@
+// SPDX-License-Identifier: LGPL-3.0-or-later
+// Copyright (C) 2025 Wouter Deconinck
+
+#pragma once
+
+#include <Acts/EventData/ProxyAccessor.hpp>
+#include <ActsExamples/EventData/Track.hpp>
+#include <ActsExamples/EventData/Trajectories.hpp>
+
+namespace eicrecon {
+
+inline ActsExamples::TrajectoriesContainer
+CreateTrajectories(const ActsExamples::ConstTrackContainer& tracks) {
```

Comment:
Should we move this to the ActsSvc?

Reply:
Right now, we don't have a standard for "helper" libraries in eicrecon. It looks like things are moving towards sprinkling header-only utility collections, so I'm looking for a better way to organize things. Also, it looks like your function can't be completely stateless, it begs for proper logging.

Reply:
Thanks for attempting this. We can move items to a follow-up, of course.

I think getting rid of the two "benchmark" is good. We also should resolve #1412, perhaps it would be best to move to JANA services only (would require porting some services from algorithms).